.github/workflows/linters.yml

```diff
@@ -0,0 +1,33 @@
+name: Linters
+
+on:
+  workflow_call:
+
+jobs:
+  black-formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Black Code Formatter
+        uses: gdubicki/black-action@main
```

Comment:
Perhaps better to use the "official" github action?

https://black.readthedocs.io/en/stable/integrations/github_actions.html

Reply:
Good point, switched to it!